main/index.js

```diff
@@ -16,14 +18,28 @@ const autoUpdater = require('./updater')
 // Prepare the renderer once the app is ready
 app.on('ready', async () => {
   await prepareNext('./renderer')
-
-  const mainWindow = new BrowserWindow({
-    width: 320,
-    height: 580,
-    resizable: true,
-    titleBarStyle: 'hiddenInset',
-    icon: join(__dirname, 'main/static/icon.icns')
-  })
+  let mainWindow
+  switch (platform()) {
+    case 'win32':
+      mainWindow = new BrowserWindow({
+        width: 320,
+        height: 580,
+        resizable: false,
+        frame: false,
+        icon: join(__dirname, 'main/static/icon.ico')
+      })
+      break;
+  
+    default:
+      mainWindow = new BrowserWindow({
+        width: 320,
+        height: 580,
+        resizable: false,
+        titleBarStyle: 'hiddenInset',
+        icon: join(__dirname, 'main/static/icon.icns')
```

Comment:
can you remove the `switch statement` and use

```js
{
 frame: os.platform() !== 'win32',
  icon: os.platform() === 'win32' ? join(__dirname, 'main/static/icon.ico') : join(__dirname, 'main/static/icon.icns')
}
```

Also, `resizable` should `true` (at least for macOS)